sunpy/net/vso/vso.py

```diff
@@ -888,3 +891,35 @@ def unknown_method(self, response):
     @classmethod
     def _can_handle_query(cls, *query):
         return all([x.__class__.__name__ in attrs.__all__ for x in query])
+
+    @classmethod
+    def register_values(cls):
+        try:
+            adict = cls.get_vso_values()
+        except URLError:
+            log.warn('Looks like the VSO server is not reachable.')
```

Comment:
I was thinking about this the other day. I suggest we cache a version of the VSO results for this, maybe update it once per release or something, so that we always return something, even if it's not the most up to date.

Reply:
How about we update the `data.cache` to serve the cached version if there is a network problem and use `data.cache` to make this request?

Reply:
That would work. Bonus points for being able to pre-populate it in the tarball.